ledger-service/http-json/src/main/scala/com/digitalasset/http/domain.scala

```diff
@@ -156,6 +156,11 @@ object domain extends com.daml.fetchcontracts.domain.Aliases {
 
   final case class UserDetails(userId: String, primaryParty: Option[String])
 
+  object UserDetails {
+    def fromUser(user: User) =
+      UserDetails(user.id, user.primaryParty.map(_.toString))
```

Comment:
Don't call toString on Strings.

```suggestion
      UserDetails(user.id, user.primaryParty)
```